numpy_financial/tests/test_financial.py

```diff
@@ -410,3 +402,27 @@ def test_broadcast_decimal(self):
                             [Decimal('-74.998201'), Decimal('-75.62318601'),
                              Decimal('-75.62318601'), Decimal('-76.88882405'),
                              Decimal('-76.88882405')], 4)
+
+
+class TestNper:
+    def test_basic_values(self):
+        assert_allclose(
+            npf.nper([0, 0.075], -2000, 0, 100000),
+            [50, 21.544944],  # Computed using Google Sheet's NPER
+            rtol=1e-5,
+        )
+
+    def test_gh_18(self):
+        with numpy.errstate(divide='raise'):
+            assert_allclose(
+                npf.nper(0.1, 0, -500, 1500),
+                11.52670461,  # Computed using Google Sheet's NPER
+            )
+
+    def test_infinite_payments(self):
+        with numpy.errstate(divide='raise'):
+            result = npf.nper(0, -0.0, 1000)
+        assert result == numpy.inf
```

Comment:
``assert`` will only work when Python is built in debug mode (``__debug__`` is true). Instead of assert, could you use NumPy's assert_equal function?

Reply:
I certainly can change it-I see that you are using Pytest as the test runner in CI, which works fine with `assert`; see e.g.

https://docs.pytest.org/en/latest/assert.html#asserting-with-the-assert-statement.

Reply:
I switched to using `numpy.testing.assert_`, though, so as not to rock the boat.

Reply:
With `pytest`, a plain `assert` is fine.  These days we use plain `assert` throughout the NumPy and SciPy test suites.

Reply:
My apologies, it looks like I'm the one who is behind. Either way it looks good now.